### DISCUSSION TITLE
OpenJVerein ohne Hibiscus nutzbar?

### DISCUSSION THREAD

**Discussion by oe1rsa** [openjverein/jverein#284] [Q&A] [answered]
Kann man OpenJVerein auch ohne Hibiscus sinnvoll nutzen, z.B. wenn die Bank keinen kompatiblen Online Zugang hat?

**Comment by dippeal** [comment]
Ich würde sagen ja. Du kannst die Buchungen auch manuell eingeben.

**Comment by oe1rsa** [comment]
Danke, zunächst mal!

Heißt das konkret, dass man Hibiscus gar nicht zu installieren braucht, oder muss man es trotzdem installieren und mit einem manuellen Konto konfigurieren?

**Comment by dippeal** [comment, marked as answer]
Wie in https://openjverein.gitbook.io/doku/allgemein/installation beschrieben muss hibiscus installiert werden. Es müssen aber keine Zugänge/Konten im hibiscus konfiguriert werden. Die Konten können im OpenjVerein (Buchführung > Konten) angelegt werden und dann verwendet werden.

**Reply by oe1rsa** [reply]
Vielen Dank. Ich werde das so versuchen.

**Comment by JohannMaierhofer** [comment]
Um Buchungen zu erzeugen gibt es dann 3 Möglichkeiten:
- durch manuelle Eingabe
- von csv Dateien importieren (Menü unter Buchführung) 
- per QIF Import bei dem die Daten von einer anderen Software exportiert wurden (Menü unter Administration Buchführung), allerdings nur für die Ersteinrichtung

**Reply by oe1rsa** [reply]
Ich habe herausgefunden, dass ich die Umsätze per csv herunterladen kann. Wenn ich nun die Kopfzeilen entsprechend (manuell) umbenenne so könnte ich die Umsätze direkt in JVerein importieren.

Andererseits konnte ich auch in Hibiscus ein offline Konto anlegen in das ich die CSV Datei direkt importieren konnte, weil Hibiscus eine komfortable Methode anbietet die Spaltenzuordnung vorzunehmen. Hat diese Vorgangsweise irgendwelche Nachteile die ich eventuell noch nicht absehen kann?

Ich bin mir bewusst, dass die Antwort von dippeal absolut korrekt auf meine Frage war, da ich ja nach einer Nutzung ohne Hibiscus gefragt hatte. Da war mir noch nicht klar, dass ich auch in Hibiscus mit cvs Import arbeiten könnte.

Ich erlaube mir deshalb meine Frage etwas auszudehnen und nach weiteren Tipps für eine optimale Verwendung ohne automatischen Bank Abgleich zu fragen.
